client/webpack.config.js

```diff
@@ -121,7 +126,7 @@ const CSS_LOADER = combineLoaders([
             sourceMap: true
         }
     },
-    { loader: 'postcss' },
+    {loader: 'postcss'},
```

Comment:
ESLint should be giving an error on this change... we typically want spaces here.